src/app/wine-modal/WineRegisterForm.tsx

```
@@ -0,0 +1,217 @@
+          <div>
+            <label
+              htmlFor='wine-image'
+              className='mb-2 block text-[1.6rem] font-bold'
+            >
+              와인 사진
+            </label>
+            <div
+              role='button'
+              tabIndex={0}
+              aria-label='와인 사진 업로드'
+              className='mb-2 flex h-[96px] w-[96px] cursor-pointer items-center justify-center rounded-[1.2rem] border border-[#ECE9DD] bg-transparent'
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)